lib/uvm/vm-execute.js

```diff
@@ -61,7 +62,8 @@
         lib = requires[i];
         // Play around with the require cache
         try {
-            libraries[lib.object] = __require(lib.require);
+            required = __require(lib.require);
+            libraries[lib.object] = lib.require === 'lodash3' ? required.noConflict() : required;
```

Comment:
Makes sense, but what if .noConflict exists for other libraries as well? Do we want to require without conflicts there as well?